### ISSUE TITLE
Publish git master on npm using "next"

### ISSUE THREAD

**Issue by romainneutron** [symfony/webpack-encore#721]
Hello,

Would it be possible to publish the latest version of the master on npm under a special name like "next" ?

React does this for instance, the publish "unstable", "canary" and "experimental", see:
https://github.com/facebook/react/releases
and https://www.npmjs.com/package/react ("versions" tab)

**Comment by Kocal** [comment]
Hi,

As a temporary workaround and if you use yarn, you can run `yarn add -D https://github.com/symfony/webpack-encore.git#master` to get the latest version.
Idk if this work with npm.

**Comment by weaverryan** [comment]
I agree with @Kocal. I don't usually (unlike currently, as I'm trying to fit a few more features in if possible) delay between merging code and releasing (unlike a large library like React which wants people to be able to test out the bleeding edge).

I'm not against it, I just don't know if it's needed - because you can use what Kocal posted.
